web/src/translation/translation.ts

```diff
@@ -4,29 +4,18 @@ export const translateIfExists = (key: string) => {
   return i18next.exists(key) ? i18next.t(key) : '';
 };
 
-export const getZoneName = (zoneCode: string) =>
-  translateIfExists(`zoneShortName.${zoneCode}.zoneName`);
+export const getFullZoneName = (zoneCode: string) => {
+  return translateIfExists(`zoneShortName.${zoneCode}.zoneName`);
+};
+export const getZoneName = (zoneCode: string) => {
+  const displayName = translateIfExists(`zoneShortName.${zoneCode}.displayName`);
```

Comment:
Have we tested that this translateIfExists works properly with the i18n system?

I can't remember if I did that or not when migrating.

Reply:
We are already using it, so I sure hope that you did test it 😅 
But should be fine, otherwise current production app would be broken

Reply:
So do I! 😅

But remember who did the thorough review 😉